pkg/util/utils.go

```diff
@@ -101,3 +107,34 @@ func (m MultiError) ToError() error {
 	}
 	return fmt.Errorf(strings.Join(errStrings, "\n"))
 }
+
+func ParseOcHelpCommand(cmdOut []byte) []string {
```

Comment:
Instead of putting this into a generic _utils_ package, we maybe should put this into a new oc package?